client/src/pages/SearchPage.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import useAsync from "../hooks/useAsync";
+import Footer from "../components/Footer";
+import styled from "@emotion/styled";
+import { useLocation } from "react-router-dom";
+import { getFilteredEvents } from "../api/getFilteredEvents";
+import EventCard from "../components/EventCard";
+import SearchPageHeader from "../components/SearchPageHeader";
+
+const Container = styled.div`
+  display: flex;
+  flex-flow: column;
+  margin-left: 0.8em;
+`;
+
+function SearchPage() {
+  const useQuery = () => {
+    return new URLSearchParams(useLocation().search);
+  };
+  let query = useQuery();
+  console.log(query);
+  const { data: events } = useAsync(getFilteredEvents, query.get("q"));
+  console.log("[SearchPage] events:", events);
```

Comment:
Do you still need these console.logs?

Reply:
Ah Jonas, no review without a redundant console.logs